src/plugins/test/accounting_test01.cpp

```diff
@@ -0,0 +1,121 @@
+/************************************************************\
+ * Copyright 2024 Lawrence Livermore National Security, LLC
+ * (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+ *
+ * This file is part of the Flux resource manager framework.
+ * For details, see https://github.com/flux-framework.
+ *
+ * SPDX-License-Identifier: LGPL-3.0
+\************************************************************/
+
+extern "C" {
+#if HAVE_CONFIG_H
+#include "config.h"
+#endif
+}
+
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <map>
+#include <string>
+
+#include "src/plugins/accounting.hpp"
+#include "src/common/libtap/tap.h"
+
+// define a test users map to run tests on
+std::map<int, std::map<std::string, Association>> users;
+std::map<int, std::string> users_def_bank;
+
+
+/*
+ * helper function to add a user/bank to the users map
+ */
+void add_user_to_map (
+                std::map<int, std::map<std::string, Association>> &users,
+                int userid,
+                const std::string& bank,
+                Association user_bank)
+{
+    // insert user to users map
+    users[userid][bank] = {
+        user_bank.bank_name,
+        user_bank.fairshare,
+        user_bank.max_run_jobs,
+        user_bank.cur_run_jobs,
+        user_bank.max_active_jobs,
+        user_bank.cur_active_jobs,
+        user_bank.held_jobs,
+        user_bank.queues,
+        user_bank.queue_factor,
+        user_bank.active
+    };
+}
+
+
+/*
+ * helper function to add a user/bank to the default bank map
+ */
+void add_user_to_def_bank_map (std::map<int, std::string> &users_def_bank,
+                               int userid,
+                               const std::string& bank)
+{
+    users_def_bank[userid] = bank;
+}
+
+
+/*
+ * helper function to add test users to the users map
+ */
+void initialize_map (
+    std::map<int, std::map<std::string, Association>> &users)
+{
+    Association user1 = {"bank_A", 0.5, 5, 0, 7, 0, {}, {}, 0, 1};
+    Association user2 = {"bank_A", 0.5, 5, 0, 7, 0, {}, {}, 0, 1};
+
+    add_user_to_map (users, 1001, "bank_A", user1);
+    add_user_to_def_bank_map (users_def_bank, 1001, "bank_A");
+
+    // purposely do not add user2 to the def_bank_map
+    add_user_to_map (users, 1002, "bank_A", user2);
+}
+
+
+static void test_basic_string_comparison ()
+{
+    const std::string string1 = "hello, world!";
+    const std::string string2 = "hello, world!";
+
+    ok (string1 == string2, "i can perform a basic string comparison test");
```

Comment:
What's this test for?

Reply:
Oh, my bad, I had written that test when I was trying to get the unit test file to actually run with `make check`. Sorry about that! I'll just remove this test altogether since it's not related to any flux-accounting stuff.